src/user/user.schema.ts

```diff
@@ -0,0 +1,17 @@
+import { Field, Int, ObjectType } from '@nestjs/graphql';
+
+@ObjectType()
+export class User {
+
+  @Field((type) => Int)
+  id: number;
+
+  @Field()
+  firstName: string;
+
+  @Field()
+  lastName: string;
+
+  @Field((type) => Int)
+  age: number;
+}
```

Comment:
@Nandita-Nikam Why are you making output type in schema file?

Reply:
@nirav-auitech i didn't get it, but i will check how we can do it differently

Reply:
output type should be in `user.types.ts`

Reply:
ok

Reply:
@nirav-auitech i deleted user.schema file because it wasn't needed and i test the graphql queries and mutations everything worked with entity file itself